week-2/Poulos - Assignment 2/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>My Portfolio</title>
+</head>
+<body>
+	<h1>Mckenna Poulos' work</h1>
+
+	<h2>Website Description</h2>
+
+	<p>This website will be a place to collectively represent my art and design work. I want to show pictures, talk about my work, and use it as a reference point for furture endevors. Ideally, I would like to include a table of contents that would link to the multiple pages in order to create and organized format. I also would like my site to be visually appealling with unique designs with a personalized layout. </p>
+
+	<h2>Ideas for pages:</h2>
+	<p>Below are my ides what what I would like to include on the other pages.</p>
+
+	<ul>
+		<li>Photo Gallery</li> <ul>
```

Comment:
It may have been a typo, but it would be nice to start this `<ul>` tag on the next line just for legibility.